### PR TITLE
Create job app

### PR DESCRIPTION
**Pull request by shayakrach**
Please merge after #25.
Depend on the user app.

### CODE REVIEW

**Review by AshkenYariv — APPROVED** [review]
Looks good!

File: jobs/tests.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+@pytest.mark.django_db
+def test_empty_database():
+    num_of_jobs = Job.objects.all().count()
+    assert num_of_jobs == 0
+
+
+@pytest.mark.django_db
+def test_job_str(new_job_generator):
+    print(new_job_generator.__str__)
```
Comment (yuvalk) [inline jobs/tests.py]: y do you print? if for debugging, plz remove this from PR

**Review by AshkenYariv — APPROVED** [review]

**Comment by yuvalk** [comment]
> Please merge after #25. Depend on the user app.

is it really dependent on that?
y?

**Comment by shayakrach** [comment]
I made a mistake.
Close this PR and move to #28.
As it does not depend on #25, the new PR does not need it.

**Pull request closed without merging**